### PR TITLE
EEP: Milestones to Embulk v1.0, and versioning strategies to follow

### PR DESCRIPTION
**Pull request by dmikurube**
The Embulk core has completed the huge architectural transition through the Embulk v0.10 "development" series.

The stable production versions, v1.0, have come into our view, but we have not been very clear about when to declare v1.0.

The goal of this EEP is to clarify the milestones for Embulk v1.0, and to suggest future versioning strategies to follow v1.0.

----

Rendered version: https://github.com/embulk/embulk/blob/eep-milestones-to-v1.0/docs/eeps/draft-milestones-to-v1.0-and-versioning-strategies.md

### CODE REVIEW

File: docs/eeps/draft-milestones-to-v1.0-and-versioning-strategies.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Embulk SPI
+------------
+
+The Embulk SPI has two-digit version numbers such as `v1.2`.
+
+Embulk core
+------------
+
+The Embulk core has three-digit version numbers such as `v1.2.3`. The versioning strategies after Embulk v1.1 would give some respects to the [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html) while it may not be very strict.
```
Comment (hiroyuki-sato) [inline docs/eeps/draft-milestones-to-v1.0-and-versioning-strategies.md:59]: What do you think to change `v1.1` -> `v1.1.*`? (Three-digit version?)

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍 (Left, One minor comment)

File: docs/eeps/draft-milestones-to-v1.0-and-versioning-strategies.md (hunk 1 above)
Comment (dmikurube) [reply docs/eeps/draft-milestones-to-v1.0-and-versioning-strategies.md:59]: Sounds reasonable!

**Comment by dmikurube** [comment]
Jfyi, it's draft yet. More updates would come, but nice to hear that it looked good. :)

**Comment by hiroyuki-sato** [comment]
Oh Draft.., I was half asleep.

**Comment by dmikurube** [comment]
This would be a big milestone. I'm wondering if I can get as many feedbacks as possible from @embulk/core-team members. Looking forward to your comments!

**Review by hiroyuki-sato — APPROVED** [review]
I'm awake. LGTM👍. I'll add more comment if something find a feedback.

**Comment by dmikurube** [comment]
We observed no concerns. Finalizing this as EEP-8 to be Accepted. (Not "Final" because we have not reached to v1.0.)

**Pull request merged**